cmake/UriParserConfig.cmake.in

```diff
@@ -0,0 +1,56 @@
+# Author: KangLin(kl222@126.com)
+
+# UriParser
+# --------
+#
+# Find UriParser
```

Comment:
Please keep spelling of uriparser all-lowercase where adressing humans like the website does and so on. If you find other spellings (that are not all-uppercase and targetting humans) somewhere, that's a bug. Thanks!